back/index.js

```diff
@@ -312,7 +312,7 @@ app.get("/children", (req, res) => {
   });
 });
 
-app.post("/children", (req, res) => {
+app.post("/children/:id", (req, res) => {
```

Comment:
why an id here? this should be generated by auto increment

Reply:
It should be family_id.. I guess

Reply:
I guess not :)